src/Avatar/Avatar.js

```diff
@@ -110,3 +135,5 @@ Avatar.defaultProps = {
 Avatar.contextTypes = {
   styleManager: PropTypes.object.isRequired,
 };
+
+Avatar.muiName = 'Avatar';
```

Comment:
At the moment it's used in Chip to determine the child type - I'll leave it until the Chip API is solidified.